src/Components/App.jsx

```diff
@@ -0,0 +1,20 @@
+import { Route, Routes } from 'react-router-dom';
+import Navbar from '../routes/navBar';
+import Books from './books';
+import Categories from './categories';
+import Header from './header';
+
+function App() {
+  return (
+    <>
+      <Header />
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="/categories" element={<Categories />} />
+      </Routes>
+    </>
+  );
+}
+
+export default App;
```

Comment:
Great job implementing the functionalities 👍 . Kindly remove the app.js from the component folder. It is supposed to be inside SRC folder and not a child folder 😊